docs/all-available-settings.edn

```diff
@@ -22,6 +22,7 @@
  :hover {:hide-file-location? false
          :arity-on-same-line? false
          :clojuredocs true}
+ :completion {:additional-edits-warning nil}
```

Comment:
Maybe make clear this should be a string?
```suggestion
 :completion {:additional-edits-warning-text nil}
```

Reply:
Yep, will change.